plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/persistence/LibraryClasspathContainerSerializer.java

```diff
@@ -46,10 +53,11 @@
  */
 @Creatable
 public class LibraryClasspathContainerSerializer {
-
   private static final Logger logger =
       Logger.getLogger(LibraryClasspathContainerSerializer.class.getName());
 
+  private static final String MASTER_LIBRARY_LIST_FILE_ID = "master-libraries";
```

Comment:
I wonder if we should name this something like `_libraries`, and rename it something like `CONTAINER_LIBRARY_LIST_FILE_ID`.  Thinking toward adding support #2400 where we currently have the App Engine container too.